ansible_collections/arista/avd/plugins/plugin_utils/eos_designs_shared_utils/wan.py

```diff
@@ -63,7 +63,9 @@ def wan_listen_ranges(self: SharedUtils) -> list:
         )
 
     @cached_property
-    def cv_pathfinder_transit_mode(self: SharedUtils) -> Literal["region", "zone"] | None:
+    def cv_pathfinder_transit_mode(
```

Comment:
Seems like this file was linted with the wrong line length

Reply:
fixed